src/app/layouts/common-layout/common-layout.module.ts

```diff
@@ -18,8 +18,12 @@ import { ProfileService } from 'src/app/services/profile/profile.service';
 import { PropertiesComponent } from 'src/app/pages/properties/properties.component';
 import { CardComponent } from 'src/app/components/card/card.component';
 import { PaymentModule } from 'src/app/pages/payment/payment.module';
+
+import { PropertyDetailsComponent } from 'src/app/pages/property-details/property-details.component';
 import { PropertyDetailsModule } from 'src/app/pages/property-details/property-details.module';
 import { DepositsComponent } from 'src/app/pages/deposits/deposits.component';
+import { OwlDateTimeModule, OwlNativeDateTimeModule } from 'ng-pick-datetime';
+import { ClientsComponent } from 'src/app/pages/clients/clients.component';
 
```

Comment:
We don't seem to be using `PropertyDetailsComponent`, `OwlDateTimeModule` and `OwlNativeDateTimeModule` anywhere in this module. Let's remove unused imports.

Reply:
Thanks, @Victoradukwu I have removed them